plugins/source/aws/resources/services/emr/studio_session_mapping.go

```diff
@@ -0,0 +1,61 @@
+package emr
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/emr"
+	"github.com/aws/aws-sdk-go-v2/service/emr/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func studioSessionMapping() *schema.Table {
+	tableName := "aws_emr_studio_session_mapping"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/emr/latest/APIReference/API_GetStudioSessionMapping.html`,
+		Resolver:            fetchEmrStudioSessionMapping,
+		PreResourceResolver: getSessionMapping,
+		Transform:           transformers.TransformWithStruct(&types.SessionMappingDetail{}, transformers.WithPrimaryKeys("StudioId", "IdentityType", "IdentityId")),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(true),
+			client.DefaultRegionColumn(true),
+		},
```

Comment:
Now that the parent table has just `arn` as the PK, you can change this PK to be just `studio_arn:IdentityType:IdentityId` and example of this can be found in the `aws_cloudwatchlogs_log_group_data_protection_policies` table

Reply:
Okay, that should now be the case in the latest commit https://github.com/cloudquery/cloudquery/pull/12529/commits/bbc2805a0afe607553ff5c333b98ba43a2d63f2f